spec/presenters/single_content_item_presenter_spec.rb

```diff
@@ -10,18 +10,6 @@
     SingleContentItemPresenter.new(current_period_data, previous_period_data, date_range)
   end
 
-  describe '#publishing_app' do
-    it 'does not fail if no publishing app' do
-      current_period_data[:metadata][:publishing_app] = nil
-
-      expect(subject.publishing_app).to eq('Unknown')
-    end
-
-    it 'capitalizes the publishing_app if present' do
-      expect(subject.publishing_app).to eq('Publisher')
-    end
-  end
-
   describe '#trend_percentage' do
     it 'calculates an increase percentage change' do
       current_period_data[:time_series_metrics] = [{ name: 'upviews', total: 100 }]
```

Comment:
Can we add tests for edit_url and edit_label?

Reply:
Sure, I've added a couple of basic tests now. Let me know what you think https://github.com/alphagov/content-data-admin/pull/132/files#diff-2495b53354dceef543a1931fbc51b5ceR71